src/commands/forget.js

```diff
@@ -42,15 +44,29 @@ export default {
 		if (length === 0) return await interaction.editReply({ content: toSay });
 
 		try {
-			await interaction?.editReply({
-				content: toSay,
-				files: [
-					{
-						attachment: log,
-						name: "context.md",
+			const request = await fetch(
+				`${process.env.WASTEBIN_HOST}/`,
+				{
+					method: "POST",
+					headers: {
+						"Content-Type": "application/json",
 					},
-				],
+					body: JSON.stringify({
+						text: log.toString("utf-8"),
+						extension: "md",
+					}),
+				},
+			)
+
+			const button = new ButtonBuilder()
```

Comment:
I've been using the term "memories" in place of "history" as to make SpongeChat seem more "alive". This could be changed to something more on-brand with such, i.e. "View memories".